bib/sofbib/software.bib

```diff
@@ -0,0 +1,8 @@
+@misc{batteryCost,
```

Comment:
I'd prefer to see this as a yaml. Bibble doesn't do great with latex as is, and I'd like to replace it in the future. There's also way much nicer linters for yaml than BibTex, which is a nice to have for new users

I do think we need at least 3+ entries here (Preferably 5+) before this goes live

Reply:
I agree that yaml is a more elegant solution. Also Bibble hasn't been updated in two years. Changing from Bibble to yaml will also resolve the below comment re: folders and bib_files.

Reply:
Also, I agree that we need more repos, maybe @venkvis or @rkurchin can suggest some that should be added

Reply:
I can add some of my stuff; do you want me to just do it in this format for now or are you going to change over to yaml or whatever?

Reply:
If you just give me a list, I'll add it so I can prototype with several repos. Thanks!